articles/downloading_data.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Downloading data"
+layout: article
+excerpt: Tutorial on how to download data from synapse with the clients 
+---
+
```

Comment:
Need overview of why and what.

Also focus the article on lessons or things the user might want to do:

Downloading a file
Downloading a directory
Managing where files are stored
....